scripts/validate_docstrings.py

```diff
@@ -691,7 +691,11 @@ def get_validation_data(doc):
 
     if doc.is_function_or_method:
         if not doc.returns:
-            if 'return' in doc.method_source:
+            if re.search(r""" \"\"\".*\"\"\".*  # Skip the docstring.
+                              \n[ \t\f\v]*return  # Find a return command.
```

Comment:
Why these specific whitespace characters and not just  \s?

Reply:
At first, I used `\s`, but was receiving this deprecation warning:

```
scripts/validate_docstrings.py:695
  /Users/danielcosta/pandas-daniel/scripts/validate_docstrings.py:695: DeprecationWarning: invalid escape sequence \s
    '\n\s*return' +
```

[re documentation](https://docs.python.org/3/library/re.html) shows that `\s` is equivalente to `[ \t\n\r\f\v]`, so I just removed the end line and carriage return (`\n` and `\r`), which are not what I intend to consume there, and left the others. Sincerely, I couldn't find out the meaning of `\f` and `\v`.

That said, I just tested using `\s` again and it seems that while using `re.VERBOSE` I don't get any more warnings. That leaves me wondering if using `\s` in regex is really discouraged by some reason or if it was just a wrongful warning that I was receiving. Maybe you can help me clarifying this.

I'll hold on with making the substitution until then.

Reply:
I found out that the deprecation warning didn't go away with the 're.VERBOSE', but with the usage of a raw string, and now I understand why it was happening in the first place.

Thus I replaced '[ \t\f\v]' for '\s'.